cli/kubectl_kadalu/install.py

```diff
@@ -39,6 +40,21 @@ def validate(_args):
 
 def run(args):
     """ perform install subcommand """
+
+    # Check if kadalu operator is present already.
+    cmd = utils.kubectl_cmd(args) + ["get", "deployments", "-nkadalu", "-ojson"]
+    resp = utils.execute(cmd)
+    data = json.loads(resp.stdout)
+    items = data["items"]
+
+    if items:
```

Comment:
Added extra condition since I was getting index out of bound when there was no namespace present.